plugins/premium/rtc.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: RTC plugin
+title_nav: RTC (beta)
+description: Real time collaboration.
+keywords: rtc
+---
+
+{% assign pluginname = "RTC" %}
+{% assign plugincode = "rtc" %}
+
+> **Note**: This plugin is only available for [{{site.cloudname}} users]({{site.pricingpage}}).
```

Comment:
This probably needs to be based off the `feature/TinyMCE_5.6` branch instead of `develop` I'd imagine? Anyways, here's a suggestion that'll use the various variables @tylerkelly13 setup for us to use:

```suggestion
{% assign beta_feature = "The RTC plugin" %}
{% assign pre-release_type = "Closed Beta" %}
{% include misc/beta-note.md %}

{{site.requires_5_6v}}<br/>
{{site.premiumplugin}}
```

Reply:
Nice didn't know we had beta headers in there already will try that out. Why is the 5.6 docs not released the community packages are out.

Reply:
The docs are for the enterprise version (with the sole exception of the changelog page). We don't have a community version at this stage.

Reply:
I should add the `Closed Beta` part won't work until https://github.com/tinymce/tinymce-docs/pull/1776 has been merged.

Reply:
Right then I don't think I need to base this of the 5.6 branch since this will probably not conflict with those changes. However the rtc plugin needs 5.6 so for the beta the users will have to use dev branch if 5.6 is not out on stable before we release the rtc beta.

Reply:
Added banners.